csm/CameraMan.py

```diff
@@ -0,0 +1,28 @@
+# import cv2
+
+class CameraMan():
+
+    def __init__(self, sourcePath):
+        self.updateVideoObject(sourcePath)
+
+    def updateVideoObject(self, sourcePath):
```

Comment:
What does this function do?

Reply:
I already implemented most of the camera work in ImageSources, would this call something from there? If so, could you get rid of capture here and leave a TODO for me to fill it in?